include/mbedtls/config_adjust_legacy_crypto.h

```diff
@@ -2,6 +2,8 @@
  * \file mbedtls/config_adjust_legacy_crypto.h
  * \brief Adjust legacy configuration configuration
  *
+ * This is an internal header. Do not include it directly.
```

Comment:
Note: https://github.com/Mbed-TLS/mbedtls/pull/9057 adds a new adjust file. If it gets merged first, we need to ensure that it follows the template that we're changing here.